tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/NLogTests.cs

```diff
@@ -37,24 +43,27 @@ public NLogTests(ITestOutputHelper output)
         }
 
         public static IEnumerable<object[]> GetTestData()
-        {
-            foreach (var item in PackageVersions.NLog)
-            {
-                yield return item.Concat(false);
-                yield return item.Concat(true);
-            }
-        }
-
-        public static IEnumerable<object[]> GetContextTestData()
         {
             var minScopeContext = new Version("5.0.0");
             var minMdlc = new Version("4.6.0");
-            var minMdc = new Version("4.0.0");
             foreach (var item in PackageVersions.NLog)
             {
-                var version = (string)item[0] == string.Empty ?
-                                  new Version("5.0.0") : // DEFAULT_SAMPLES
-                                  new Version((string)item[0]);
+                Version version;
+                var defaultSamples = (string)item[0] == string.Empty;
+                if (defaultSamples)
+                {
+                    // LogsInjection.NLog uses different versions depending on framework
+                    version = EnvironmentHelper.IsNetFramework() ?
+                                  new Version("2.1.0") :
+                                  new Version("5.0.0");
```

Comment:
nit: you can use the existing method here
```suggestion
                    version = EnvironmentHelper.IsCoreClr() ?
                                  new Version("5.0.0") :
                                  new Version("2.1.0");
```